server/index.js

```diff
@@ -0,0 +1,149 @@
+const express = require( 'express' ); // remember to install your npm packages
+const helmet = require( 'helmet' );
+const cors = require( 'cors' );
+const mongoose = require( 'mongoose' );
+const bodyparser = require( 'body-parser' );
+const session = require( 'express-session' );
+const axios = require( 'axios' );
```

Comment:
axios not needed on the server, unless your server is going to query some other server for data...